finagle-redis/src/main/scala/com/twitter/finagle/Redis.scala

```diff
@@ -51,9 +51,7 @@ object Redis extends Client[Command, Reply] with RedisRichClient {
       .insertBefore(DefaultPool.Role, RedisPool.module)
   }
 
-  /*
-   * ServiceFactory loaded with Command and its Reply to Stacks.
-   */
+  // Review 1
```

Comment:
`ServiceFactory` loaded with `Command` and its `Reply` to `Stack`.

-> finagle/finagle-core/src/main/scala/com/twitter/finagle/Stack.scala

> Stacks represent stackable elements of type T